db/migrate/20220608184748_create_orders.rb

```diff
@@ -0,0 +1,9 @@
+class CreateOrders < ActiveRecord::Migration[6.1]
+  def change
+    create_table :orders do |t|
+      t.bigint :employee_id
```

Comment:
you miss fk and indexes here
<img width="809" alt="Screenshot 2022-06-12 at 08 51 37" src="https://user-images.githubusercontent.com/5034850/173221095-40fd68db-1b21-4aaa-9d5f-4a81d9c2dc18.png">

Reply:
Thanks. Corrected ;)